radicle-cli/src/commands/merge.rs

```diff
@@ -144,10 +141,27 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
     //
     // Get patch information
     //
-    let patch_id = options.id.resolve(&repository.backend)?;
+    let revision_id = options.revision_id.resolve(&repository.backend)?;
+    let (patch_id, patch) = patches.find_by_revision(&revision_id)?.ok_or(anyhow!(
+        "no open patch with revision `{}` could be found",
+        &revision_id
+    ))?;
+    if !patch.is_open() {
+        anyhow::bail!(
+            "revision's patch must be open for merging, but it is `{}`",
+            patch.state()
+        );
+    }
+    let (last_revision_id, _) = patch.latest().expect("patch has one or more revisions");
```

Comment:
We don't use `expect` that way, please use `unwrap()` plus a "safety" comment, or return the error with `?`. For this case, it's possible for a patch not to have a latest, so it would be better to return an error.